.circleci/config.yml

```diff
@@ -1,29 +1,32 @@
 version: 2.1
+
 orbs:
   cfa: continuousauth/npm@1.0.2
-jobs:
-  test:
-    docker:
-      - image: cimg/node:14.17
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - v1-dependencies-{{ checksum "yarn.lock" }}
-            - v1-dependencies-
-      - run: npx yarn install
-      - save_cache:
-          paths:
-            - node_modules
-          key: v1-dependencies-{{ checksum "yarn.lock" }}
-      - run: npx yarn prettier:check
-      - run: npx yarn build
-      - run: npx yarn test
+  node: electronjs/node@1.4.1
+
 workflows:
   test_and_release:
     # Run the test jobs first, then the release only when all the test jobs are successful
     jobs:
-      - test
+      - node/test:
+          name: test-<< matrix.executor >>-<< matrix.node-version >>
+          test-steps:
+            - run: yarn prettier:check
+            - run: yarn build
+            - run: yarn test
+          use-test-steps: true
+          matrix:
+            alias: test
+            parameters:
+              executor:
+                - node/linux
+                - node/macos
+                - node/windows
+              node-version:
+                - '20.5'
```

Comment:
Sanity check: a few other recent PRs mention not using versions greater than `20.2.0`, is `20.5` fine here?

https://github.com/electron/asar/pull/278/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R30-R32

Reply:
Thanks for the check. 🙂 Yes, 20.5 is fine now with the latest version of the orb, v1.4.1. CircleCI updated their Windows image and the orb picked it up in https://github.com/electron/node-orb/pull/6. The latest orb version also supports version strings without the patch version, so I'm making use of that as well - striking balance between CI randomly breaking from a minor version change in Node.js (happened recently actually) and picking up patch releases for security fixes.

The linked config PR was just created before those changes and wasn't updated. 😅 Will pick it up the next time we're bumping versions.